File: Python/set2/Q8.py
# 8.  Write a program to solve this equation ((a+b)/(x+y))^(1/(x+y))

import math

a,b,x,y=3,6,9,2
n=a+b
d=x+y
eq=n/d
p=1/(x+y)
sol=math.pow(eq,p)
print(f"Solution: {sol:.2f}")

File: Java/set5/Q8.java
// Write a program to illustrate the use of static member function.

class A
{
    static void display()
    {
        System.out.println("This is a static member function");
    }
}
class Q8
{
    public static void main(String[]args)
    {
        A a=new A();
        a.display();
    }
}